server/controllers/lists.js

```diff
@@ -0,0 +1,71 @@
+const ToDoList = require("../models/ToDoList");
+const ToDo = require("../models/ToDo");
+const {StatusCodes} = require("http-status-codes");
+const {BadRequestError, NotFoundError} = require("../errors");
+
+const getAllLists = async (req, res) => {
+  const lists = await ToDoList.find({owner: req.user.userId}).sort("createdAt");
+  res.status(StatusCodes.OK).json({lists, count: lists.length});
+};
+
+const getList = async (req, res) => {
+  const {
+    user: {userId},
+    params: {id: listId},
+  } = req;
+  const list = await ToDoList.findOne({
+    _id: listId,
+    owner: userId,
+  });
+  if (!list) throw new NotFoundError(`No entry found with ID no. ${listId}`);
```

Comment:
Redundant validation checking, see previous comments from auth.js.

Reply:
Please elaborate.

Reply:
This item was resolved via a separate comment on auth.js file, line 22.